pkg/store/dbstore/store.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	log = ctrl.Log.WithName("store")
+	log = ctrl.Log.WithName("store -> dbstore")
```

Comment:
why use `store -> dbstore` here?

Reply:
Because I think `store -> dbstore` can be more clearly than `store` when debugging because the `dbstore` package is under the `store`.